src/notifications/background/storage.ts

```diff
@@ -0,0 +1,47 @@
+import { FeatureStorage } from '../../search/search-index-new/storage'
+
+export default class NotificationStorage extends FeatureStorage {
+    constructor(storageManager) {
+        super(storageManager)
+
+        this.storageManager.registerCollection('notifications', {
+            version: new Date(2018, 7, 4),
+            fields: {
+                id: { type: 'string' },
+                title: { type: 'string' },
+                message: { type: 'string' },
+                buttonText: {type: 'string'},
+                link: {type: 'string'},
+                sentTime: {type: 'datetime'},
```

Comment:
One more thing I forgot to bring up yesterday:
I'm not sure if these should be `datetime` field types or some kind of number type (there is `int` and `float` types introduced in #429 for integer and floating point numbers, respectively), given that in the rest of the code you're setting these field values to epoch timestamps (represented as JS `number` types). Currently I think the only use of `datetime` field types are in the `directLinks` collection schema, where that particular field is being set using JS `Date` object values:
https://github.com/WorldBrain/Memex/blob/develop/src/direct-linking/background/storage.ts#L31

Maybe @ShishKabab has a better idea of what should be done here?

Reply:
Mmm, seems this requires a bit more thought. Happy to revise this API. Depends on 1) what kind of queries we want to exectute on datetime fields and what the most convenient/performant way of storing that is, and 2) how people actually use retrieving/storing datetime fields and if it's convenient to have a datetime/moment field or not.

Reply:
Here we need only check for the existence of the `readTime` field.
@ShishKabab One more thing regarding `sentTime`, when we will make new notif, we have to see the current time and insert that with the notif.
Currently when we actually insert the data, then after I am inserting that time into the localStorage and check before every insertion.
That make sense?

Here: https://github.com/WorldBrain/Memex/pull/440/files#diff-bc224a7d660ebe3986166dfa5d9ace84R65

Reply:
Your link only points me to the schema definition  :(  Anyway, as for sentTime of static notifs, I'd say this would be the time when we release then update.

Reply:
How could we differentiate sentTime and deliveredTime then?

Reply:
@ShishKabab ?